index.html

```diff
@@ -29,320 +30,454 @@
+            hesitate to contact me.
+          </p>
+          <h3 class="connect">LET'S CONNECT</h3>
+          <ul class="social">
+            <li>
+              <a href="#" class="icons-link">
+                <img
+                  src="images/icon_image/github.png"
+                  alt="Github"
+                  class="github"
+              /></a>
             </li>
-            <li class="frame-lang">
-              <img
-                src="images/2nd-phase/html.png"
-                height="48"
-                width="48"
-              /><span class="tab-text">HTML</span>
+            <li class="icons-li">
+              <a href="#" class="icons-link">
+                <img
+                  src="images/icon_image/linkedin.png"
+                  alt="linkedin"
+                  class="linkedin"
+              /></a>
             </li>
-            <li class="frame-lang">
-              <img src="images/2nd-phase/css.png" height="48" width="48" /><span
-                class="tab-text"
-                >CSS</span
-              >
+            <li class="icons-li">
+              <a href="#" class="icons-link">
+                <img
+                  src="images/icon_image/angelist.png"
+                  alt="Hand"
+                  class="angelist"
+              /></a>
             </li>
-          </div>
-        </li>
-        <li class="language-title unopened-title">
-          <span class="language-words">Frameworks</span
-          ><span class="material-symbols-outlined"> chevron_right </span>
-        </li>
-        <li class="language-title unopened-title">
-          <span class="language-words">Skills</span>
-          <span class="material-symbols-outlined"> chevron_right </span>
-        </li>
-      </ul>
+            <li class="icons-li">
+              <a href="#" class="icons-link">
+                <img
+                  src="images/icon_image/twitter.png"
+                  alt="twitter"
+                  class="twitter"
+              /></a>
+            </li>
+            <li class="icons-li">
+              <a href="#" class="icons-link">
+                <img src="images/icon_image/medium.png" alt="Hand" class="medium"
+              /></a>
+            </li>
+          </ul>
+          <button class="resume-button">Get my resume</button>
+        </div>
+        <div class="about-half-sect">
+          <ul class="about-list">
+            <li class="tab-item tab-item-open">
+              <div class="language-title">
+                <span class="language-text">Languages </span>
+                <button class="drop-down"><img src="images/chevron.png" /></button>
+                <!-- <i class="fas fast ">&#xf078;</i> -->
+              </div>
+              <ul class="tab-langs">
+                <li class="frame-lang">
+                  <img src="images/2nd-phase/js.png" height="48" width="48" /><label
+                    class="tab-text"
+                    >JavaScript</label
```

Comment:
- [ ] Dear @Kwamay, |'m really impressed with what you did already, I wish I could have approved this, but according to the requirement `below the technology, the names are expected to be aligned to the left`. Please align your technology names to the left. 

### Figma template
![image](https://github.com/Kwamay/Porfolio_Setup_Project/assets/69077061/13bfeea9-c85d-4d03-afc1-c9f9dce3db58)

### Your design 
![image](https://github.com/Kwamay/Porfolio_Setup_Project/assets/69077061/776ad3c4-05ea-499f-983c-693d2a41c9b8)

- [ ] Also, please consider removing commented codes from your code base. Remember, if you ever wanna take a look at the code you wrote at some point you can use your commit history.